### ISSUE TITLE
Make it into a Linux system daemon on Ubuntu Linux so it starts up on reboot

### ISSUE THREAD

**Issue by jayhohoho2019** [greghesp/assistant-relay#223]
**Is your feature request related to a problem? Please describe.**
Need to run the startup process when OS is rebooted

**Describe the solution you'd like**
Install script to install necessary files to start up process at OS bootup time.

**Describe alternatives you've considered**
Added startup script to user startup list but it requires user to login first. 

**Additional context**

**Comment by 6uhrmittag** [comment]
afaik this is already built in, but not documented. I'm not sure how I found out about it... anyway:

In the `assistant-relay` directory, run:
````cli
pm2 startup
````
and it should output a command. Run it as `root` to setup&enable the service(default name is `pm2-root`).

The Documentation is pretty straightforward: https://pm2.keymetrics.io/docs/usage/startup/

I set this up a few month ago, on a raspberry and never touched it since than.

Pls check if it works - maybe it's just a simple addition to the docs
